docker-compose.yml

```diff
@@ -107,3 +107,7 @@ services:
       uniregistrar_driver_did_ethr_ethrNetworkMetaPublicKeys: ${uniregistrar_driver_did_ethr_ethrNetworkMetaPublicKeys}
     ports:
       - "9080:3000"
+  driver-did-bid:
+    image: caictdevelop/driver-did-bid:latest
```

Comment:
I think this is the wrong Docker image, or?

Maybe it should be `caictdevelop/uni-registrar-driver-did-bid:latest` instead?

Reply:
name update completed